### DISCUSSION TITLE
The nested query is not as expected

### DISCUSSION THREAD

**Discussion by myNameIsDu** [prisma/prisma#14297] [Q&A] [answered]
<img width="1486" alt="image" src="https://user-images.githubusercontent.com/53305259/178961538-34ecff22-a210-4cb2-831b-6804afcf0389.png">

I want to update all the component records that it related this page record, Then it gave some error, But the document is showed it works  

https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#update-an-existing-user-record-by-updating-the-profile-record-its-connected-to

**Comment by myNameIsDu** [comment]
I solved  this problem many with `updateMany` , But I'm still curious why I can't use the examples in the document

**Comment by austincrim** [comment, marked as answer]
Hey there!

Can you share your Prisma schema?

**Reply by myNameIsDu** [reply]
Yeah, It looks like this

```prisma
model CmsPage {
  id                                 Int                                  @id @default(autoincrement())
  title                              String                               @db.VarChar(50)
  description                        String?                              @db.VarChar(300)
  cmsPageComponent                   CmsPageComponent[]
  @@map("cms_page")
}

model CmsPageComponent {
  id                                 Int                                  @id @default(autoincrement())
  status                             String                               @db.VarChar(10)
  pageId                             Int
  elementId                          String                               @db.VarChar(50)
  exist                              Boolean                              @default(true)
  page                               CmsPage                              @relation(fields: [pageId], references: [id])
  @@index([pageId], map: "cms_page_component_pageId_fkey")
  @@map("cms_page_component")
}
```

**Reply by austincrim** [reply]
The example you linked to in the docs is updating a 1-to-1 relation, your code is updating a 1-to-many relation. So, using `updateMany` is the right choice here.

**Reply by myNameIsDu** [reply]
Oh. I understood it. Thank you so much for your help.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
